Gruntfile.js

```diff
@@ -25,6 +25,14 @@ const ELECTRON_PACKAGE_JSON = 'electron/package.json';
 const PACKAGE_JSON = 'package.json';
 const INFO_JSON = 'info.json';
 
+const DESKTOP_SPEC = {
+  "Version": "1.1",
```

Comment:
What version is that and where it appears?

Reply:
Maybe we can just use the `'<%= info.version %>'` (as in other places).

Also use single quotes for strings.. `'`

Reply:
It is the version of the desktop entry specification: https://standards.freedesktop.org/desktop-entry-spec/1.1/apfs02.html

Reply:
Right.. :)

Reply:
Also mentioned in the description... I'm blind :D